docs/release_notes/dec2023.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 1
+id: December_2023
+description: Release notes for December
+title: December 2023
+tags:
+  - release notes
+  - changelog
+---
+
+## 3.2.6 (December 18, 2023)
+
+- Prophecy Python libs version: 1.7.03
+- Prophecy Scala libs version: 7.1.46
+
+### Features {#FeaturesRelease326}
+
+#### Gem Dependencies
+
+In our latest 3.2 Release, we've empowered users with the ability to build their own Gems. This allows you to incorporate custom logic essential for your specific needs directly into the Gem.
+Frequently, this involves utilizing custom dependencies within the Gems for tasks such as validations and transformations.
+With this release, we've introduced a feature that enables you to seamlessly integrate any custom Pypi or Maven dependency into a Gem during the build process.
+This dependency is also automatically added into the Pipeline using the Gem whenever needed. Stay tuned for more comprehensive documentation on this enhancement.
+
+#### Build Template configurable by Admin
+
+Cluster administrators now have the capability to modify the build template for their organization by navigating to the Admin settings. By accessing the 'Build System' tab under the 'Admin' section, administrators can update the template, ensuring that any newly created projects will adopt the latest template. Additionally, existing projects will transition to the updated template, provided the template hasn't been customized.
+
+![build_templates](img/3.2.6_build_template.png)
+
+#### Minor Improvements
+
+- **Task Group Support in Airflow**: We have added support for Taskgroup in Low code Airflow. Airflow task groups are a tool to organize tasks into groups within your DAGs. This is useful to organize complicated DAGs by visually grouping tasks that belong together. This is also useful when you have to apply some `default_args` to sets of tasks, instead of at the DAG level.
```

Comment:
This is also useful when you have to apply some `default_args` to a set of tasks, instead of applying at the DAG level.